src/dmd/backend/dwarfdbginf.d

```diff
@@ -2817,11 +2821,57 @@ static if (1)
     const(char)* getSymName(Symbol* sym)
     {
         version (MARS)
-            return sym.prettyIdent ? sym.prettyIdent : sym.Sident.ptr;
+            return getNameWithoutModule(sym.prettyIdent ? sym.prettyIdent : sym.Sident.ptr);
         else
             return sym.Sident.ptr;
     }
 
+    /**
+     *  Returns the correct name of a symbol.
+     *
+     *  Params:
+     *      ident = name of the symbol
+     *
+     *  Returns:
+     *      The symbol's name without the module's name as prefix
+     */
+    extern(D) const(char)[] getNameWithoutModule(const(char)[] ident)
```

Comment:
This overload serves no purpose, as it is only called by the other `getNameWithoutModule`